css/my_portfolio.css

```diff
@@ -451,3 +451,448 @@ button:active {
+  .list-style-none {
+    list-style: none;
+  }
+
+  .text-align-center-lg {
+    text-align: center;
+  }
+
+  .align-items-center-lg {
+    align-items: center;
+  }
+
+  .d-none-lg {
+    display: none;
+  }
+
+  .d-flex-lg {
+    display: flex;
+  }
+
+  .d-block-lg {
+    display: block;
+  }
+
+  .d-grid-lg {
+    display: grid;
+  }
+
+  .pos-absolute-lg {
+    position: absolute;
+  }
+
+  .width-40-lg {
+    width: 40%;
+  }
+
+  .margin-right-23-lg {
+    margin-right: 23%;
+  }
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    background: #3c3a39;
+    padding: 2%;
+    align-items: center;
+    position: fixed;
+    z-index: 1;
+  }
+
+  #navbar {
+    margin-right: 10%;
+  }
+
+  .nav-link {
+    color: #fff;
+    text-decoration: none;
+    margin-right: 2em;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .my-self {
+    background: #1c1a19;
+    padding: 20% 5%;
+    height: 150vh;
+  }
+
+  .myself-header {
+    width: 40%;
+    position: absolute;
+    left: 11%;
+    top: 44%;
+  }
+
+  .my-name {
+    font-family: "Crete Round", sans-serif;
+    color: #ff6b00;
+    font-weight: normal;
+    font-size: 56px;
+    line-height: 64px;
+  }
+
+  .group2 {
+    background-image: url("../assets/images/group2.svg");
+    width: 41em;
+    height: 47em;
+    left: 43em;
+    top: 0.6em;
+  }
+
+  .media-icons {
+    position: absolute;
+    top: 52%;
+  }
+
+  .margin-top-lg {
+    margin-top: 1em;
+  }
+
+  .my-text {
+    width: 80%;
+  }
+
+  .my-work {
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .card-work {
+    margin-top: 10%;
+  }
+
+  .cards {
+    margin-top: 10%;
+    display: flex;
+    margin-left: 5%;
+  }
+
+  .card-header {
+    display: flex;
+  }
+
+  .card-header-name {
+    color: #091e42;
+    font-family: 'Crete Round', sans-serif;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    margin-left: 10%;
+  }
+
+  .card-header-border {
+    width: 50%;
+    border: 1px solid #212121;
+    margin: 2.5%;
+  }
+
+  .card-text {
+    text-align: left;
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    line-height: 28px;
+    color: #091e42;
+  }
+
+  .story {
+    width: 33%;
+  }
+
+  .card-name {
+    margin-left: 0;
+  }
+
+  .skills {
+    width: 95%;
+  }
+
+  .skill1 {
+    margin: 4% 1% 4% 4%;
+    border: 1px solid #8993a4;
+    width: 4em;
+    background: rgba(255, 255, 255, 0.24);
+    padding: 0;
+  }
+
+  .skill2 {
+    margin: 4% 2%;
+    border: 1px solid #8993a4;
+    width: 4em;
+    background: rgba(255, 255, 255, 0.24);
+    padding: 0;
+  }
+
+  .skill3 {
+    margin: 4% 1% 4%;
+    border: 1px solid #8993a4;
+    width: 8em;
+    background: rgba(255, 255, 255, 0.24);
+    padding: 0;
+  }
+
+  .skill4 {
+    margin: 4% 1% 4% 1%;
```

Comment:
- [ ] It would be nice to use use the `color: #091e42` for the `.skill1-skill4` declarations.

Reply:
Thanks.
Added